docs/standard/containerized-lifecycle-architecture/docker-devops-workflow/docker-application-outer-loop-devops-workflow.md

```diff
@@ -43,191 +43,169 @@ When using Docker for the deployment, the "final artifacts" to be deployed are D
+Typically, most Docker applications are composed of multiple containers rather than a single container. A good example is a microservices-oriented application for which you would have one container per microservice. But, even without strictly following the microservices approach patterns, it's probable that your Docker application would be composed of multiple containers or services.
 
 Therefore, after building the application containers in the CI pipeline, you also need to deploy, integrate, and test the application as a whole with all of its containers within an integration Docker host or even into a test cluster to which your containers are distributed.
 
 If you're using a single host, you can use Docker commands such as docker-compose to build and deploy related containers to test and validate the Docker environment in a single VM. But, if you are working with an orchestrator cluster like DC/OS, Kubernetes, or Docker Swarm, you need to deploy your containers through a different mechanism or orchestrator, depending on your selected cluster/scheduler.
 
 Following are several types of tests that you can run against Docker containers:
 
--   Unit tests for Docker containers
+- Unit tests for Docker containers
 
--   Testing groups of interrelated applications or microservices
+- Testing groups of interrelated applications or microservices
 
--   Test in production and "canary" releases
+- Test in production and "canary" releases
 
-The important point is that when running integration and functional tests, you must run those tests from outside of the containers. Tests must not be defined and run within the containers that you are deploying, because the containers are based on static images that should be exactly like those that you will be deploying into production.
+The important point is that when running integration and functional tests, you must run those tests from outside of the containers. Tests are not contained or run in the containers you are deploying, because the containers are based on static images that should be exactly like the ones you'll be deploying to production.
 
-A very feasible option when testing more advanced scenarios like testing several clusters (test cluster, staging cluster, and production cluster) is to publish the images to a registry to test in various clusters.
+A practical option when testing more advanced scenarios, like including several clusters (test cluster, staging cluster, and production cluster) is to publish the images to a registry, so it can be tested in various clusters.
 
 ### Push the custom application Docker image into your global Docker Registry
 
 After the Docker images have been tested and validated, you'll want to tag and publish them to your Docker registry. The Docker registry is a critical piece in the Docker application life cycle because it is the central place where you store your custom test (aka "blessed images") to be deployed into QA and production environments.
```

Comment:
aka is bad for loc 

```suggestion
After the Docker images have been tested and validated, you'll want to tag and publish them to your Docker registry. The Docker registry is a critical piece in the Docker application life cycle because it is the central place where you store your custom test (also known as "blessed images") to be deployed into QA and production environments.
```